### PR TITLE
fix wrong session language in classroom-ladder

### PR DESCRIPTION
**Pull request by smallst**
since classroom-ladder page's url don't contains `course-instanceid`, so we use `league` as session query url

### CODE REVIEW

File: app/lib/LevelLoader.coffee [hunk 1]
```diff
@@ -48,6 +48,7 @@ module.exports = class LevelLoader extends CocoClass
     @observing = options.observing
     @courseID = options.courseID
     @courseInstanceID = options.courseInstanceID
+    @league = utils.getQueryVariable 'league'
```
Comment (codeluggage) [inline app/lib/LevelLoader.coffee:51]: Does this need to be here, or could it be done where it is used?

**Review by codeluggage — COMMENTED** [review]
Seems fine to me but I am not very familiar with this code :)

File: app/lib/LevelLoader.coffee [hunk 2]
```diff
@@ -159,6 +160,8 @@ module.exports = class LevelLoader extends CocoClass
       url = "/db/level/#{@levelID}/session"
       if @team
         url += "?team=#{@team}"
+        if @level.isType('course-ladder') and @league and not @couseInstanceID
```
Comment (nwinter) [inline app/lib/LevelLoader.coffee:163]:
```suggestion
        if @level.isType('course-ladder') and @league and not @courseInstanceID
```

Comment (nwinter) [reply app/lib/LevelLoader.coffee:163]: Also, is `@team` always set in these cases, or do we sometimes need to handle the `else if @courseID` branch below?

**Review by nwinter — CHANGES_REQUESTED** [review]
Took a quick look.

File: app/lib/LevelLoader.coffee (hunk 2 above)
Comment (smallst) [reply app/lib/LevelLoader.coffee:163]: @nwinter hey, i think `@team` will be always set since the bug comes from `play as red` or `play as blue` button.

**Review by nwinter — APPROVED** [review]

**Pull request merged**